sample.env

```diff
@@ -103,4 +103,8 @@ AEM_LOG_TO_FILE=false
 AEM_LOG_LEVEL=INFO
 ADM_LOG_TO_FILE=false
 ADM_LOG_TO_CONSOLE=true
-ADM_LOG_LEVEL=INFO
\ No newline at end of file
+ADM_LOG_LEVEL=INFO
+
+# ODE Monitoring
+ODE_TIM_INGEST_MONITORING_ENABLED=true
+TIM_INGEST_MONITORING_INTERVAL=60
```

Comment:
When I run the solution locally using Docker Compose, I encounter a NumberFormatException in the TimDepositController constructor. This happens because the ODE_TIM_INGEST_MONITORING_INTERVAL environment variable contains an empty string. The environment variable in sample.env has a different name than the one referenced in docker-compose.yml. Either the environment variable in sample.env should be renamed to ODE_TIM_INGEST_MONITORING_INTERVAL, or the reference in docker-compose.yml should be updated. We should also consider handling the empty string case in the code.

Reply:
I made the changes to the sample.env [here](https://github.com/CDOT-CV/jpo-ode/pull/106/commits/4554878a8a7e33f7e1402cd77ed1635169d4d155) and I'm protecting from the empty string case [here](https://github.com/CDOT-CV/jpo-ode/pull/106/commits/4554878a8a7e33f7e1402cd77ed1635169d4d155)